### DISCUSSION TITLE
Recovering Account

### DISCUSSION THREAD

**Discussion by DixonKH** [community/community#135270] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I suddenly deleted existed github account last year so is it possible to recover that deleted github account again?

**Comment by ManojPaloi** [comment]
response #1

**Comment by ManojPaloi** [comment]
response #2

**Comment by eabdalmufid** [comment]
> I suddenly deleted existed github account last year so is it possible to recover that deleted github account again?

It is not possible to recover a deleted GitHub account. Once an account has been deleted, all data associated with that account is permanently removed from GitHub's servers. If you would like to continue using GitHub, you will need to create a new account.

**Reply by resanwilda** [reply]
👍🏻
